lib/team_statistics.rb

```diff
@@ -0,0 +1,109 @@
+module TeamStatistics
+
+  def team_info(team_id)
+    found_team = @teams.fetch(team_id.to_i)
+    {
+      "team_id" => found_team.team_id.to_s,
+      "franchise_id" => found_team.franchise_id.to_s,
+      "team_name" => found_team.team_name,
+      "abbreviation" => found_team.abbreviation,
+      "link" => found_team.link
+    }
+  end
+
+  def best_season(team_id)
+    team_id = team_id.to_i
+    filtered_games = []
+    games_by_team = @games.each do |game_id, game|
```

Comment:
We aren't calling the games_by_team variable anywhere in the best_season method so we can remove it.